prover/lang/kore-lang.md

```diff
@@ -161,6 +161,7 @@ only in this scenario*.
                    | "partial" "(" Patterns ")"
                    | "heap" "(" Sort "," Sort ")" // Location, Data
                    | "\\hole" "(" ")" [klabel(Phole)]
+                   | "\\functionalPattern" "(" Pattern ")"
```

Comment:
Add the comment "\functionalPattern(phi) is a syntactic sugar of \exists x . phi = x".

Reply:
Done.